tests/cupy_tests/test_cudnn.py

```diff
@@ -333,6 +334,11 @@ def setUp(self):
         if ((self.dilate > 1 and version < 6000) or
                 (self.groups > 1 and version < 7000)):
             self.err = ValueError
+        elif (sys.platform.startswith('win32') and version == 7605
+                and deterministic and self.dtype == numpy.float16
+                and self.ndim == 3 and self.dilate == 2 and self.groups == 2):
```

Comment:
This is a very stringent skip condition, just enough to skip the failing tests in the Windows CI. I checked cuDNN's documentation and release notes, but I don't think a Windows-only failure like this is expected, so I'd prefer to make the skip range narrow. If we expand the Windows CI to test more versions, we will have a better clue and can revisit this.